.github/workflows/build.yml

```diff
@@ -328,3 +332,51 @@ jobs:
       - name: Upload release
         run: |
           aws s3 sync --acl public-read dist s3://static.realm.io/ci/test/${{ env.PACKAGE_VERSION }}/
+      - name: Publish Github Release
+        uses: ncipollo/release-action@3ac4132803a6419fa2a7f4e9dbd1d93fceb690b9
+        with:
+          artifacts: "dist/*.exe,dist/*.dmg,dist/*.zip,dist/*.AppImage"
+          bodyFile: dist/RELEASENOTES.md
+          name: ${{ steps.get-version.outputs.version }}
+          commit: main
+          tag: ${{ steps.get-version.outputs.version }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          draft: false
+      - name: Update Changelog
+        run: |
+          echo "## vNext (TBD)
+
+          ### Enhancements
```

Comment:
I'm surprised this won't add a lot of indention into the changelog 🤔

Reply:
Indent up until the beginning of the echo is syntactical yaml indent, which is stripped off before this gets thrown into the shell - so for the shell that block doesn't have any indents at all.

Reply:
Awesome, I didn't know that.